pkg/sql/distsqlrun/flow.go

```diff
@@ -47,6 +48,8 @@ type FlowID struct {
 type FlowCtx struct {
 	log.AmbientContext
 
+	stopper *stop.Stopper
```

Comment:
Yes. I think it belongs in flowCtx, because other processors might want to have a handle on the stopper as well (e.g. to listen for a signal to exit early).